deploy/okd/manifests/0.7.0/22-packageserver.yaml

```diff
@@ -76,8 +76,8 @@ metadata:
+  tls.crt: "LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURWakNDQWo2Z0F3SUJBZ0lCQVRBTkJna3Foa2lHOXcwQkFRc0ZBREFjTVJvd0dBWURWUVFERXhGd1lXTnIKWVdkbExYTmxjblpsY2kxallUQWVGdzB4T0RBNU1Ua3hOVEk1TWpOYUZ3MHhPVEE1TVRreE5USTVNak5hTUJreApGekFWQmdOVkJBTVREbkJoWTJ0aFoyVXRjMlZ5ZG1WeU1JSUJJakFOQmdrcWhraUc5dzBCQVFFRkFBT0NBUThBCk1JSUJDZ0tDQVFFQTNwOU54MVN3d0thZHd2b014ZDJGQitwSkloMnAyK0JTMFRGVExoRit6cllrc0VuM0lJeWoKaDZ3NzB6Rk95a3VuTThGZDM5NWJOMUxUYmdNdEl5TE5LMzhMaTA3UHVwTlJFc1QwZU8rSU9OT0dyUXZCTDFQZApqZjB0WEFGOXN3REpqNkMzUVBvYTJ2MWd4dTU2eXpqREtyRjFhQXVHczhCSGhPNFJXazg2b1Y3dy9nbWpyQzNYCjMxazV3RHJrTzN6V2dzWmt0MXJUdkhnZjlwWGtFZG1uNHphKzY2N2NJSlFQMm1UeC9YazU5a1M0RDZiU0R3UU8KRTRPbTQwMVhxN2Z0bGh6NGJVLytDTEs3WmpGY05GZWJxNnVmVkxLQXFETENTTDVqK04vb0VBRVlZK01Banc5dApsVTZVUEQ3bWZYNGJUK2h4UjZuNXdhczFOck9sUHpFcVV3SURBUUFCbzRHbE1JR2lNQTRHQTFVZER3RUIvd1FFCkF3SUZvREFkQmdOVkhTVUVGakFVQmdnckJnRUZCUWNEQVFZSUt3WUJCUVVIQXdJd0RBWURWUjBUQVFIL0JBSXcKQURCakJnTlZIUkVFWERCYWdpbHdZV05yWVdkbExYTmxjblpsY2k1dmNHVnlZWFJ2Y2kxc2FXWmxZM2xqYkdVdApiV0Z1WVdkbGNvSXRjR0ZqYTJGblpTMXpaWEoyWlhJdWIzQmxjbUYwYjNJdGJHbG1aV041WTJ4bExXMWhibUZuClpYSXVjM1pqTUEwR0NTcUdTSWIzRFFFQkN3VUFBNElCQVFCL2hYYWh2NEI4Y0I2b0pQTUFqQW9tRGNyeXlINWUKb0pGN0NzaEdkSnZxQjJFYjhLK3BnMDJCdWhtdUE5WFRHcjRtcnVWeU1MNm1ZRGdGOUYzKzhlazg5bStTR052eQo4ZkMxekJHa0Q0RWJaWE9XbkZOdzhPNlI3MktBMXBrTFBNdXpBZ0pPNkMyd0piVFlVbXc2SXdPTFpzMkp2SXZnCmdSR2VEWkRybWEvSGVwZEZnLzNzaHVhQ09JRXhYd2dNRFlJWmF5RlNmb1owWGQzaldxZVlFTWY4MjY3Y1JtcEEKeDRleFZDQWhtbXlocDJ3UUNvYzdvVmZ1dDBRTlNqYkhIazNaNVROUmhWeDQ2VGNYVWQ3Y1hTNjd3K1pOczNmcAo1bWRob3o1b2tDUjVWZGdCQlU0aU5xT0FtZkUyWFl4WkZwYlJSM2tSeDBjZjIwOXMwTW53VWtSbgotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg=="
```

Comment:
Worried about these expiring :/. This really does need to be bootstrapped by OLM to get around this issue.